src/main/java/com/codeplanks/home360/listing/ListingController.java

```diff
@@ -54,18 +54,20 @@ public ResponseEntity<SuccessDataResponse<Listing>> getListing(@PathVariable Str
   }
 
   @GetMapping("/listings/search")
-  public ResponseEntity<SuccessDataResponse<PaginatedResponse<Listing>>> getFilterListings(
+  public ResponseEntity<SuccessDataResponse<PaginatedResponse<Listing>>> getMyFilterListings(

```

Comment:
You can do better with the method name, try make it a little bit more descriptive.

Reply:
Method renamed.